pkg/api/pod/util.go

```diff
@@ -44,35 +41,40 @@ const (
 // AllContainers specifies that all containers be visited
 const AllContainers ContainerType = (InitContainers | Containers | EphemeralContainers)
 
+// DefaultContainers returns a ContainerType mask which includes all container
+// types except for the ones guarded by feature gate.
+func DefaultContainers() ContainerType {
```

Comment:
I like this approach. A name like `FeatureEnabledContainers()` might be more intuitive

Reply:
named as `AllFeatureEnabledContainers`